bin/consul-templaterb

```diff
@@ -17,34 +17,52 @@ def compute_default_output(source)
 end
 
 options = {
+  vault: {
+    debug: {
+      network: false
+    },
+    base_url: ENV['VAULT_ADDR'] || 'http://localhost:8200',
+    token: ENV['VAULT_TOKEN'] || nil,
+    token_renew: true,
+    retry_duration: 10,
+    lease_duration_factor: 0.5, # The time it waits before actualizing data based on the lease time: 2h lease * 0.5 = Fetch data every 1h
+    min_duration: 60,
+    max_retry_duration: 86_400, # Much higher than consul's because some dynamic secrets could be used for a day.
+    paths: {
+      '/v1/sys/mounts' => {
+        max_retry_duration: 600,
+        min_duration: 600
+      }
+    }
+  },
   consul: {
     debug: {
       network: false
     },
     base_url: ENV['CONSUL_HTTP_ADDR'] || 'http://localhost:8500',
     token: ENV['CONSUL_HTTP_TOKEN'] || nil,
-    retry_duration: 10,      # On error, retry after n seconds
-    min_duration: 5,         # On sucess and when differences are found
-    retry_on_non_diff: 3,    # On success but when there are not differences
-    wait_duration: 600,      # Delay to block in Consul
+    retry_duration: 10, # On error, retry after n seconds
+    min_duration: 5, # On sucess and when differences are found
+    retry_on_non_diff: 3, # On success but when there are not differences
+    wait_duration: 600, # Delay to block in Consul
     max_retry_duration: 600, # On consecutive errors, delay will increase, max value
     missing_index_retry_time_on_diff: 15, # On endpoints without X-Consul-Index => next request
     missing_index_retry_time_on_unchanged: 60, # Endpoints with X-Consul index and no diff
     paths: {
-      '/v1/catalog/services': {
```

Comment:
Same, do not change this, avoid too many changes, you patch will be easier to review